config/routes.rb

```diff
@@ -40,13 +41,12 @@
         member do
           post 'auto_provision'
           put 'reboot'
+          put 'refresh_facts'
         end
         collection do
           post 'facts'
           post 'auto_provision_all'
-        end
-        member do
-          put 'refresh_facts'
+          put  'reboot_all'
```

Comment:
I didn't remove it, I moved it above the collection into the already existing `member do`